### PR TITLE
more patches for hevc dpb

### PR DESCRIPTION
**Pull request by xuguangxin**

### CODE REVIEW

**Comment by xuguangxin** [comment]
@fulinjie , please help review. thanks

**Comment by fulinjie** [comment]
Hi @xuguangxin, this seems to break fate:

04:12:43,027 INFO  - tests/Makefile:250: recipe for target 'fate-hevc-conformance-BUMPING_A_ericsson_1' failed
04:12:43,027 INFO  - make: *** [fate-hevc-conformance-BUMPING_A_ericsson_1] Error 1
04:12:43,028 INFO  - [33mwarn: "make fate V=2 KEEP=1" failed - exitcode 2[0m

**Comment by xuguangxin** [comment]
@fulinjie fixed, please help review again.
thanks

File: patchset/0032-lavc-hevc-do-not-let-missing-ref-frames-invovle-in-d.patch [hunk 1]
```diff
@@ -0,0 +1,86 @@
+From 03b78ae246dd27602af4def7c6c3372c0a0cf2a0 Mon Sep 17 00:00:00 2001
+From: Xu Guangxin <guangxin.xu@intel.com>
+Date: Mon, 16 Mar 2020 03:51:40 -0400
+Subject: [PATCH 32/37] lavc: hevc, do not let missing ref frames invovle in
+ dpb process
+
+We will generate frames for missed reference. the frame just used for refrence.
```
Comment (fulinjie) [inline patchset/0032-lavc-hevc-do-not-let-missing-ref-frames-invovle-in-d.patch:7]: Nit: just used for refrence -> reference

Comment (xuguangxin) [reply patchset/0032-lavc-hevc-do-not-let-missing-ref-frames-invovle-in-d.patch:7]: thanks, will fix by https://github.com/intel-media-ci/ffmpeg-cartwheel/pull/11

**Pull request merged**